src/diffusers/models/modeling_utils.py

```diff
@@ -937,7 +935,6 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                             offload_folder=offload_folder,
                             offload_state_dict=offload_state_dict,
                             dtype=torch_dtype,
-                            force_hooks=force_hook,
```

Comment:
These are not needed because we actually removes all hooks and dispatch the model again in the pipeline logic.

Reply:
If the model were to be used independently of the pipeline, would removing this be sensible?

Reply:
Only if the user used device_map with only one gpu. Should be fine to be honest. The user actually expect to be able to move the model without any issues if the model in dispatch on only one gpu.